packages/cm6-graphql/__tests__/test.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jest/expect-expect */
```

Comment:
I guess this file is never run, because it's missing `{test,spec}` suffix, what do we need to do with it?

Reply:
I think this has since been renamed to `test.spec.ts` on `main`